lib/src/ui/common/widget/primary_icon_button.dart

```diff
@@ -20,7 +20,7 @@ class PrimaryIconButton extends StatelessWidget {
   Widget build(BuildContext context) => PrimaryButton(
         onPressed: onPressed,
         child: Padding(
-          padding: EdgeInsets.all(_Constants.iconPadding),
+          padding: const EdgeInsets.all(_Constants.iconPadding),
```

Comment:
w sumie skoro to const to całe może polecieć do constów

Reply:
Tutaj się zastanawiam, bo w przypadku UI będziemy mieli dość dużo constów z uwagi na optymalizację, tak więc myślę kiedy przenosić to gdzieś do osobnej klasy z constami, a kiedy nie. Mamy przykładowo:

```
const Center(child: CircularProgressIndicator());

const BoxConstraints();

const EdgeInsets.symmetric(
            horizontal: _Constants.paddingHorizontal,
            vertical: _Constants.paddingVertical,
          );

const SizedBox(height: _Constants.sectionHeaderPadding);

const IconThemeData(color: Colors.black);

const EdgeInsets.only(
            top: _Constants.appBarActionsVerticalPadding,
            bottom: _Constants.appBarActionsVerticalPadding,
            right: Dimens.pagePadding,
            left: _Constants.appBarActionsIconsPadding,
          );

const Text(UntranslatableStrings.newYorkTimes);
```

Stąd też nie wiem czy nie zostać przy tym jak mamy, czyli do stałych wrzucamy surowe wartości, tj. liczby && stringi, a te UI flutter specific zostawić tam gdzie UI, tylko widzę, że mamy już gdzieś w stałych np. `Color(0x2DD5D5D5)` - jeżeli trzymać się jakiejś konwencji to wolałbym trzymać samą wartość tj. 0x2DD5D5D5, aniżeli przenosić wszystko do Constants, a jeżeli elastycznie to możemy ustalić, że np. liczby && stringi && kolory i może ikony (?) do constants, reszta w kodzie. Jak się na to zapatrujesz? :)

Reply:
Jestem za 
> liczby && stringi && kolory

ikony mogą zostać w widgetach to będzie bardziej przejrzyście imo, po lewej stronie przy numerze linii wtedy pokazuje się ta ikonka